rasa/core/processor.py

```diff
@@ -290,11 +290,22 @@ def _log_slots(tracker):
         if slot_values.strip():
             logger.debug(f"Current slot values: \n{slot_values}")
 
+    def _get_keywordlist():
+        keywordlist = ["restart", "back", "stop", "start", "action"]
+        return keywordlist
+
     def _log_unseen_intent(self, parse_data: Dict[Text, Any]) -> None:
-        """check if the NLU picks up intent that aren't in the domain.
+        """check if the NLU picks up intent that aren't in the domain and intent shouldn't be the valid default intents or self domain intents in the domain.
         """
         intent = parse_data["intent"]["name"]
-        if intent and self.domain and intent not in self.domain.intents:
+
+        if intent in self._get_keywordlist() or intent in self.domain.intents:
+            logger.error(
+                "Interpreter parsed an intent '{}' "
+                "that is should not defined in the domain.".format(intent)
```

Comment:
We don't want to log an error for this, these should be silent if the intent is in the list of keywords, because the intent is valid

Reply:
But if we don't want to show any warning or error then there is no point to track those keywords in intents input.
Please suggest.

Reply:
The idea is that if those keywords match the intent, we don't show the `"Interpreter parsed an intent '{}' "that is not defined in the domain.` error. therefore the logic below would look something like 
```
if intent and self.domain and intent not in self.domain.intents and intent not in default_intents:
```
so that if it matches a keyword that warning will not show. does that make sense?

Reply:
Probably makes sense to even make its own variable to keep things readable, like 
```
intent_is_recognized = intent and (intent in self.domain.intents or intent in default_intents)
```
so that you can easily do
```
if not intent_is_recognized and not domain_is_empty:
```
for the warning

Reply:
Okay, I will implement this.